modules/mapping-utils/src/resolveSets.js

```diff
@@ -2,40 +2,36 @@ import { get, isEmpty, uniq } from 'lodash';
 import uuid from 'uuid/v4';
 import { buildQuery } from '@arranger/middleware';
 
-const mapIds = ({ response, path }) =>
-  response.hits.hits.map(x => get(x, `_source.${path.split('__').join('.')}`));
-
 const retrieveSetIds = async ({
   es,
   index,
   type,
   query,
   path,
-  SCROLL_TIME = '1m',
   BULK_SIZE = 1000,
 }) => {
-  const handleResult = async ({ scrollId, total, ids = [] }) => {
-    if (ids.length === total) return uniq(ids);
-    const response = await es.scroll({ scroll: SCROLL_TIME, scrollId });
-    return handleResult({
-      total,
-      scrollId: response._scroll_id,
-      ids: [...ids, ...mapIds({ response, path })],
+  const search = async ({ searchAfter }) => {
```

Comment:
does it make more sense to add the default here instead of passing an empty object at the call spot?

```
async ({ searchAfter } = {}) =>
```

Reply:
yup